Tools/scripts/build-jsbsim.sh

```diff
@@ -6,16 +6,18 @@ echo "---------- $0 start ----------"
 
 # don't waste time rebuilding jsbsim if we already have a working copy of it
 # this can save ~10 or minutes on some hardware
-JSBBINARY=~/jsbsim/src/JSBSim
+JSBBINARY=~/jsbsim/build/src/JSBSim
 if [ -f $JSBBINARY ]; then
     echo "$JSBBINARY exists, skipping build. (to force rebuild then remove this file and re-run) "
 else
     echo "$JSBBINARY does not exist, building it in your home folder:"
     cd ~ 
     rm -rf jsbsim
-    git clone https://github.com/tridge/jsbsim.git
+    git clone git://github.com/JSBSim-Team/jsbsim.git
     cd jsbsim
-    ./autogen.sh
+    mkdir build
+    cd build
```

Comment:
cmake -DCMAKE_CXX_FLAGS_RELEASE="-O3 -march=native -mtune=native" -DCMAKE_C_FLAGS_RELEASE="-O3 -march=native -mtune=native" -DCMAKE_BUILD_TYPE=Release ..
as stated on wiki, as I doubt that we will modify it

Reply:
You are completely right. Changed it accordingly.